RestAPI.cs

```diff
@@ -43,12 +43,12 @@ public class RestAPI
             if (string.IsNullOrEmpty(url))
                 throw new Exception("Please use a valid WooCommerce Restful API url.");
 
-            url = url.Trim().ToLower().TrimEnd('/');

```

Comment:
Oh, this is a pull request, i did fix the issue and added the code on github.  Requesting you merge it into your main branch.